app/apps/show/operations/controller.js

```diff
@@ -1,5 +1,6 @@
 import Ember from 'ember';
+import Paginated from 'diesel/mixins/controllers/paginated';
```

Comment:
relative imports are generally preferred, as they avoid sprinkling the app name around and make the code more portable. At least, this has been the idiom so far.